db/pre-re/pet_db.conf

```diff
@@ -27,7 +27,7 @@
 //= Pets Database
 //=========================================================================
 
```

Comment:
Can you see my Replay ? 
bug ? 
[ReplayPetevol.zip](https://github.com/HerculesWS/Hercules/files/2081375/ReplayPetevol.zip)
```
petevo.rrf = [Poring => Mastering]
petevo2.rrf = [petevo.rrf > Relog  > get petegg > use Pet Incubator > My pet is disappear [Mastering ] 
petevo3.rrf = [petevo2.rrf > Relog  > get petegg > use Pet Incubator > My pet is disappear]
```

Reply:
@sctnightcore that is the result of missing pet infos in the database, there is no delay on hunger and initial intimacy is zero, so the pet leaves you once you hatch it.

Reply:
:thinking: